build.gradle

```diff
@@ -167,8 +162,7 @@ subprojects {
     repositories {
         mavenLocal()
         mavenCentral()
-        maven { url = uri("http://nexus.phx.connexta.com:8081/nexus/content/repositories/ion-releases/") }
-        maven { url = uri("http://nexus.phx.connexta.com:8081/nexus/content/groups/public/") }
+        maven { url "http://nexus.phx.connexta.com:8081/nexus/content/repositories/ion-releases/" }
```

Comment:
I clean out my gradle cache and performed a clean & build with gradle with this URL, which is also used in the Ion Ingest API project. Gradle did not complain about missing repository information. Now both repos use the same nexus URL.